apps/src/sites/studio/pages/maker/setup.js

```diff
@@ -111,15 +121,15 @@ const BoardSetupStatus = React.createClass({
      */
     function forEachLedInSequence(func, delay) {
       return new Promise(resolve => {
-        const leds = bc.prewiredComponents.colorLeds;
+        const leds = board.prewiredComponents_.colorLeds;
```

Comment:
So, I'm torn here between just giving the board a `celebrate()` method that will do all of this stuff internally, and finding a nicer way to expose its components (which are what we usually interact with anyway).  The only reason this hasn't already come up is that we inject the components into the interpreter where they're normally used.

I'm also considering making all of the components public members on CircuitPlaygroundBoard at some point, which would solve this problem.